src/function_wrappers.ts

```diff
@@ -25,9 +25,9 @@ import {
   Context,
   CloudEventFunction,
   CloudEventFunctionWithCallback,
-  CloudEventsContext,
   HandlerFunction,
 } from './functions';
+import {CloudEvent} from 'cloudevents';
```

Comment:
you should use the interface not the concrete class implementation. Also you should probably just pull this from `./functions` where it is re-exported:

```typescript
  // ...
  CloudEvent,
} from './functions';
```